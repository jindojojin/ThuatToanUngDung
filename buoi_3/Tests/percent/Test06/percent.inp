7
5 80 5 10 0
25 50 20 0 5
15 50 5 30 0
0 70 10 20 0
5 30 15 0 50
10 50 0 0 40
0 0 0 0 100
