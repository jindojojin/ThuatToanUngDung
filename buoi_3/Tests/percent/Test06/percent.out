20
20
20
10
20
10
1
